src/spikeinterface/sortingcomponents/motion_interpolation.py

```diff
@@ -17,7 +17,13 @@
 
 
 def correct_motion_on_peaks(
-    peaks, peak_locations, times, motion, temporal_bins, spatial_bins, direction="y", progress_bar=False
+    peaks,
+    peak_locations,
+    times,
+    motion,
+    temporal_bins,
```

Comment:
it would be useful to clarify whether these are bin edges or bin centers

Reply:
I think it would be nice if estimate_motion returned an object so that this function could take fewer arguments, and to avoid replicating the logic about temporal bins and motion interpolation in several places (here in peaks and in traces and anywhere else a user may want to get displacement at a particular depth and time)

maybe a class like this one? https://github.com/evarol/dredge/blob/main/dredge-python/dredge/motion_util.py#L22

Reply:
This is bin center this is in the docstring no ?
I am agree for the object but this will be in the next round of modification that we will to togoether no ?